src/main/java/com/epam/ta/reportportal/dao/AuthConfigRepositoryCustomImpl.java

```diff
@@ -49,32 +51,30 @@ public AuthConfigRepositoryCustomImpl(EntityManager entityManager) {
 		//		createDefaultProfileIfAbsent();
 	}
 
-	//
-	//    @Override
-	//    public void deleteSettings(AuthIntegrationType type) {
-	//        mongoOperations.updateFirst(findDefaultQuery(), new Update().unset(type.getDbField()), AuthConfigEntity.class);
-	//    }
-	//
-	//    @Override
-	//    public void updatePartially(AuthConfigEntity entity) {
-	//        mongoOperations.updateFirst(findDefaultQuery(), updateExisting(entity), AuthConfigEntity.class);
-	//    }
-	//
-	//    @Override
-	//    public void updateLdap(LdapConfig ldapConfig) {
-	//        mongoOperations
-	//                .updateFirst(findDefaultQuery(), Update.update(AuthIntegrationType.LDAP.getDbField(), ldapConfig),
-	//                        AuthConfigEntity.class);
-	//
-	//    }
-	//
-	//    @Override
-	//    public void updateActiveDirectory(ActiveDirectoryConfig adConfig) {
-	//        mongoOperations
-	//                .updateFirst(findDefaultQuery(),
-	//                        Update.update(AuthIntegrationType.ACTIVE_DIRECTORY.getDbField(), adConfig),
-	//                        AuthConfigEntity.class);
-	//    }
+	@Transactional
+	@Override
+	public void deleteSettings(Integration integration) {
+		entityManager.remove(integration);
+	}
+
+
+	@Transactional
+	@Override
+	public void updatePartially(AuthConfig authConfig) {
```

Comment:
i guess there is no need for this method anymore. The idea was to change only those fields which are not NULL. Will it work with merge function?